src/languages/en.js

```diff
@@ -944,7 +944,8 @@ export default {
         editor: {
             nameInputLabel: 'Name',
             nameInputHelpText: 'This is the name you will see on your workspace.',
-            nameIsRequiredError: 'You need to define a name for your workspace',
+            nameIsRequiredError: 'You need to define a name for your workspace.',
+            nameHasHtml: 'HTML tags are not allowed in workspace names.',
```

Comment:
I wonder how many of these errors we're going to have... most of our inputs are not allowed to have HTML 😬

Reply:
Maybe we need something more generic where we prevent HTML in all inputs? 😅

Reply:
I'm confused. Are you wanting to check for html in more fields? 

Or do you want to make this error message more generic so that it can be used in more places?

Reply:
I am saying this bug exists in more fields, eg first and last name, but not only : it exists in most fields in our app.

What I am suggesting is that instead of waiting for the next 42 bug reports to say the same thing, we should fix this in all text inputs that don't accept HTML (which is probably 99.999% of them).

Reply:
qq, @flodnv are you suggesting it to be a follow up PR or a part of this issue?

Reply:
I think either is fine, but why not this PR, because this bug doesn't really matter much. What matters more is to fix it holistically instead of 42 times.

Reply:
I think thats fine. I'm worried this will just get dragged on the larger the scope is. Rather than just doing it for this now, and the next time this is an issue. 

The author DRY's this logic up so it can be used in two places. Then each subsequent time, people can add that method. 

But we can drop it in the issue

Reply:
Discussed here: https://github.com/Expensify/App/issues/12268#issuecomment-1405266063

We will tackle the validation and testing/QA in this GH: https://github.com/Expensify/App/issues/14610